tests/installation/reboot_after_installation.pm

```diff
@@ -103,6 +104,11 @@ sub run {
     }
     else {
         power_action('reboot', observe => 1, keepconsole => 1, first_reboot => 1);
+        # exclude remote backend where no grub_test follows reboot_after_installation and exclude test with
+        # encrypted partition as well because encrypted bootloader is handled via grub_test and it takes long time.
+        # it requires also to exclude USBBOOT because reboot after installation will come back to
+        # bootloader of installationn medium and it breaks then grub_test or first_boot
+        stop_grub_timeout unless (is_remote_backend || get_var('ENCRYPT') || get_var('USBBOOT'));
```

Comment:
Keep in mind that we want to remove all exceptional cases.

Task 2.: reboot_after_installation always left the system for a following module, in a known system state, which is not changing over time.

"The next module has to take over from grub grub_test" is just a suggestion, but task also includes other modules.

Reply:
we want to remove all exceptional cases, this is not possible at moment. we should do step by step.